lib/git_reflow/commands/setup.rb

```diff
@@ -25,5 +25,12 @@
     GitReflow::Config.add "constants.minimumApprovals", ask("Set the minimum number of approvals (leaving blank will require approval from all commenters): "), local: reflow_options[:project_only]
     GitReflow::Config.add "constants.approvalRegex", GitReflow::GitServer::PullRequest::DEFAULT_APPROVAL_REGEX, local: reflow_options[:project_only]
     
+    say("Enter the follow information associated with your remote repository: (The information is used to run 'git reflow deliver'.")
```

Comment:
We are already using the `github_api` gem.  See the [GitHelpers.remote_user](https://github.com/reenhanced/gitreflow/blob/master/lib/git_reflow/git_helpers.rb#L12) and [GitHelpers.remote_repo_name](https://github.com/reenhanced/gitreflow/blob/master/lib/git_reflow/git_helpers.rb#L17) for how we're using those to [search for existing pull requests](https://github.com/reenhanced/gitreflow/blob/master/lib/git_reflow.rb#L115).  [GitReflow.git_server](https://github.com/reenhanced/gitreflow/blob/master/lib/git_reflow.rb#L187) will give you an instance you can use to interface with the current git server's API.